src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.SearchResult.cs

```diff
@@ -17,7 +16,7 @@ private class SearchResult : INavigateToSearchResult
         {
             public string AdditionalInformation => _lazyAdditionalInfo.Value;
             public string Name => _declaredSymbolInfo.Name;
-            public string Summary => _lazySummary.Value;
+            public string Summary { get; }
```

Comment:
Yup.  BUt INavigateToSearchResult is an interface that we still use for TS.  I don't want to change it and have to worry about that right now.  Probably after 15.3/ask mode.